### PR TITLE
Garands have a 1% chance to give you Garand thumb.

### PR DESCRIPTION
**Pull request by Naaanii**
## About The Pull Request
Garands have a 1% chance to deal 1 brute to the hand the gun is in.
## Why It's Good For The Game
Muh realism.
Meh gacha mechanic but interesting and funny implementation.
Feel the PVT in you come out with the 1% chance to smack your thumb.
image
## Changelog
:cl:
add: Added Garand thumb mechanic. (1% chance to deal 1 brute when reloading a Garand)
/:cl:

### CODE REVIEW

**Comment by Oscari-pog** [comment]
Additional 1% chance to the already 1% chance that makes it tear off your entire arm

**Comment by ilikesaying** [comment]
should also apply painscream

File: code/modules/projectiles/gun_system.dm [hunk 1]
```diff
@@ -1570,6 +1570,18 @@
 	var/obj/item/ammo_magazine/magazine = mag
 	return magazine?.bonus_overlay
 
+/obj/item/weapon/gun/rifle/garand/reload(obj/item/new_mag, mob/living/user, force = FALSE)
+	. = ..()
+	if(!.)
+		return
+	if(prob(1))
```
Comment (Lumipharon) [inline code/modules/projectiles/gun_system.dm:1577]: you need to check for user

Comment (Naaanii) [reply code/modules/projectiles/gun_system.dm]: Done

**Comment by Naaanii** [comment]
> should also apply painscream

Done

**Comment by Naaanii** [comment]
> Additional 1% chance to the already 1% chance that makes it tear off your entire arm

Would be funny but alas

File: code/modules/projectiles/gun_system.dm [hunk 2]
```diff
@@ -1570,6 +1570,19 @@
...
 	return magazine?.bonus_overlay
 
+/obj/item/weapon/gun/rifle/garand/reload(obj/item/new_mag, mob/living/user, force = FALSE)
+	. = ..()
+	if(!.)
+		return
+	if(user && prob(1))
+		garand_thumb(user)
+
+/obj/item/weapon/gun/rifle/garand/proc/garand_thumb(mob/living/user)
```
Comment (TiviPlus) [inline code/modules/projectiles/gun_system.dm:1580]:
```suggestion

///Gets yout thumb stuck in the gun while reloading
/obj/item/weapon/gun/rifle/garand/proc/garand_thumb(mob/living/user)
```

**Pull request merged**